pkg/skaffold/deploy/kubectl.go

```diff
@@ -42,10 +42,11 @@ import (
 type KubectlDeployer struct {
 	*latest.KubectlDeploy
 
-	originalImages     []build.Artifact
-	workingDir         string
-	kubectl            deploy.CLI
-	insecureRegistries map[string]bool
+	originalImages       []build.Artifact
+	workingDir           string
+	kubectl              deploy.CLI
+	insecureRegistries   map[string]bool
+	debugHelpersRegistry string
```

Comment:
This new `debugHelpersRegistry` seems out of place here and suggests we need to somehow improve how we communicate information to the debug transforms.

Reply:
I replaced the `debugHelpersRegistry` field with `globalConfig`. I'm then pulling the `debugHelpersRegistry` property from the global config in `renderManifests`. This allowed me to drop the additional `debugHelpersRegistry` field in `runcontext.RunContext`

I think the `globalConfig` field is also more appropriate than `debugHelpersRegistry` but not sure if it's good enough